Description: Sorting collected source files. Bug #828788.
Author: Reiner Herrmann <reiner@reiner-h.de>
        (the patch provider)
Reviewed-by: Herbert Parentes Fortes Neto <hpfn@debian.org>
Last-Update: 2016-06-28
Index: pyparted-3.10.7/setup.py
===================================================================
--- pyparted-3.10.7.orig/setup.py
+++ pyparted-3.10.7/setup.py
@@ -73,7 +73,7 @@ setup(name='pyparted',
       packages=['parted'],
       package_dir={'parted': 'src/parted'},
       ext_modules=[Extension('_ped',
-                             glob.glob(os.path.join('src', '*.c')),
+                             sorted(glob.glob(os.path.join('src', '*.c'))),
                              define_macros=features,
                              **pkgconfig('libparted',
                                          include_dirs=['include']))
